src/pages/AppConfigure.vue

```diff
@@ -13,8 +15,17 @@ function switchTab(tab: ConfigureTab) {
 
 <template>
   <div class="container">
```

Comment:
We may not need the outermost `div`. Can we put the container class on the `VStack`.